config/commons.Makefile.j2

```diff
@@ -30,8 +30,12 @@ $(MAPPINGS_DIR)/{{ mapping.local_name }}: | $(MAPPINGS_DIR)/
 	wget {{ mapping.mirror_from|default(mapping.mapping_set_id) }} -O $@
 {% endfor %}
 
-mappings: $(ALL_MAPPINGS)
+.PHONY: biomappings
+biomappings: | $(MAPPINGS_DIR)/
+	python3 scripts/process_biomappings.py
```

Comment:
The commons.Makefile.j2 should not be edited this way:

YOu should

1. Add the mapping you want to create to https://github.com/monarch-initiative/monarch-mapping-commons/blob/main/registry.yaml
2. Run https://github.com/monarch-initiative/monarch-mapping-commons/blob/main/update_registry.sh to rebuild the makefile
3. Add the biomappings goals in https://github.com/monarch-initiative/monarch-mapping-commons/blob/main/custom.Makefile, which is the custom extension of the autogenerated Makefile.